torch/csrc/distributed/rpc/script_call.cpp

```diff
@@ -9,11 +9,13 @@ const std::string ScriptCall::ATEN_PREFIX_("aten::");
 
 ScriptCall::ScriptCall(
     std::shared_ptr<Operator> op,
+    // NOLINTNEXTLINE(cppcoreguidelines-rvalue-reference-param-not-moved)
     std::vector<at::IValue>&& stack)
     : op_(std::move(op)), stack_(stack), isAsyncExecution_(false) {}
```

Comment:
Why not move these parameters here? Seems like pretty clearly should be a move into `stack_`?

Reply:
A previous commit using std::move here caused test core dump, it needs further investigation.

Reply:
@cyyever Sounds like that is probably caused by a use after free wherever the ctor is called?

Reply:
I think so.